cli_helpers/tabular_output/preprocessors.py

```diff
@@ -77,21 +77,22 @@ def align_decimals(data, headers, **_):
 
     :param iterable data: An :term:`iterable` (e.g. list) of rows.
     :param iterable headers: The column headers.
+    :param iterable column_types: The columns' type objects (e.g. int or float).
     :return: The processed data and headers.
     :rtype: tuple
 
     """
     pointpos = len(headers) * [0]
     for row in data:
         for i, v in enumerate(row):
-            if isinstance(v, Decimal):
+            if column_types[i] is float and isinstance(v, Decimal):
```

Comment:
Why only `isinstance(v, Decimal)`? Don't we want to align other float types, i.e. `type(v) in float_types`?